src/main/java/com/baeldung/spring/MvcConfig.java

```diff
@@ -107,4 +109,8 @@ public Validator getValidator() {
         return validator;
     }
 
+    @Bean
+    WebServerFactoryCustomizer<ConfigurableServletWebServerFactory> enableDefaultServlet() {
```

Comment:
Why not public?

Reply:
This bean will not be used outside, Needed this only to enable the default servlet which is required at the start of the application.